pkg/route/validation/validation_test.go

```diff
@@ -1200,6 +1854,120 @@ func TestValidateRouteUpdate(t *testing.T) {
 	}
 }
 
+// This unit test tests the regex for validating various values which a user can pass.
+func TestHeaderValues(t *testing.T) {
+	// The syntax is http-request set-header <name> <fmt> [ { if | unless } <condition> ]. As per  As per this http://cbonte.github.io/haproxy-dconv/2.6/configuration.html#4.2-http-request%20add-header
```

Comment:
Conditional formatting is a non-goal in the EP.
```suggestion
	// The syntax is http-request set-header <name> <fmt>. As per  As per this http://cbonte.github.io/haproxy-dconv/2.6/configuration.html#4.2-http-request%20add-header
```

Reply:
as mentioned we will allow if condition but it will work string and not be interpreted by HAProxy.

Reply:
to avoid confusion we can remove this.

Reply:
https://github.com/openshift/library-go/commit/398bba91d32aa80169a697d8b06ba0ff5edc5774#diff-51cbcb9e560e512d7dba0cd93dab386055814a27e2c7903ad93bce92c58fd18bR1858